recipe/meta.yaml

```diff
@@ -37,25 +39,27 @@ requirements:
     - setuptools >=18.5
 
 # generally failing
-{% set nose_exclude = "recursion" %}
+{% set nose_exclude = "recursion|test_set_matplotlib_formats|test_set_matplotlib_close" %}
 {% set nose_exclude = nose_exclude ~ "|test_history" %}  # [not win]
 # https://github.com/ipython/ipython/issues/12164
 {% set nose_exclude = nose_exclude ~ "|system_interrupt" %}  # [linux]
 {% set nose_exclude = nose_exclude ~ "|home_dir_3|home_dir_5|store_restore|storemagic" %}  # [win]
 
 test:
   requires:
-    - curio  # [unix]
-    - ipykernel
-    - matplotlib  # [python_impl == 'cpython']
     - nbformat
     - nose >=0.10.1
-    - numpy
-    - pygments
+    - pip
     - requests
     - testpath
+    # would _ideally_ be in the block below, but messes with re-rendering
+    - numpy
+    {% if not building_on_new_platform %}
```

Comment:
We need to check if we can do test downstreams instead: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#downstream-tests